docs/self-managed/concepts/multi-tenancy.md

```diff
@@ -0,0 +1,86 @@
+---
+id: multi-tenancy
+title: "Multi-tenancy"
+sidebar_label: "Multi-tenancy"
+description: "Multi-tenancy allows you to re-use your Camunda installation."
+---
+
+:::caution
+Multi-tenancy is disabled by default and can be enabled by the use of environment variables. This feature should be
+enabled in all required components, see:
+
+- [Identity feature flags](../../../self-managed/identity/deployment/configuration-variables/#feature-flags)
+- [Zeebe multi-tenancy](../../../self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy)
+- [Operate multi-tenancy](../../../self-managed/operate-deployment/operate-configuration/)
+- [Tasklist multi-tenancy](../../../self-managed/tasklist-deployment/tasklist-configuration/#multi-tenancy)
+- [Optimize multi-tenancy]($optimize$/self-managed/optimize-deployment/configuration/multi-tenancy/)
+- [Connectors multi-tenancy](../../../self-managed/connectors-deployment/connectors-configuration/#multi-tenancy)
+  :::
+
+## Multi-tenancy in Camunda 8
+
+Multi-tenancy in the context of Camunda 8 refers to the ability of the Camunda 8 platform to serve multiple distinct
+tenants or clients within a single installation. Multi-tenancy in Camunda 8 extends these capabilities to cater to the
+needs of different departments, teams, or even external clients, all within a shared Camunda environment. Here's a closer
+look at what multi-tenancy is in Camunda 8:
+
+### Isolation of Data and Processes
```

Comment:
We just need to ensure all of these headers are sentence-case, so "Isolation of data and processes", "Resource sharing", etc.